spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.11/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala211DataprocImage13DisableConscryptAcceptanceTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.acceptance;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+public class Scala211DataprocImage13DisableConscryptAcceptanceTest
```

Comment:
Do we want to add tests for unsupported Dataproc image versions as well? https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#unsupported_dataproc_versions

Reply:
Yes, some customers still use it